news/2371-rattler-build-bld-bat-lint.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Added lint rule to flag `bld.bat` files when using rattler-build. rattler-build uses `build.bat` instead of `bld.bat` for Windows builds (#2371).
```

Comment:
```suggestion
* Added lint rule to flag `bld.bat` files when using rattler-build. rattler-build uses `build.bat` instead of `bld.bat` for Windows builds (#2372).
```